src/main/java/org/sagebionetworks/bridge/workerPlatform/bridge/BridgeUtils.java

```diff
@@ -1,13 +1,33 @@
 package org.sagebionetworks.bridge.workerPlatform.bridge;
 
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
 import org.sagebionetworks.bridge.rest.model.App;
 import org.sagebionetworks.bridge.rest.model.Exporter3Configuration;
+import org.sagebionetworks.bridge.rest.model.Study;
+import org.sagebionetworks.bridge.rest.model.StudyPhase;
 
 public class BridgeUtils {
+    private static final Set<StudyPhase> EXPORTABLE_PHASES = ImmutableSet.of(StudyPhase.LEGACY, StudyPhase.RECRUITMENT,
+            StudyPhase.IN_FLIGHT);
```

Comment:
Allow exports in Analysis and Completed. This means if someone accidentally flips to those states, we don't have ot manually redrive their exports. (Add a warning if we're in this state so that we can follow up as needed.)